resources/views/components/modal.blade.php

```diff
@@ -3,24 +3,13 @@
 @php
 $id = $id ?? md5($attributes->wire('model'));
 
-switch ($maxWidth ?? '2xl') {
-    case 'sm':
-        $maxWidth = 'sm:max-w-sm';
-        break;
-    case 'md':
-        $maxWidth = 'sm:max-w-md';
-        break;
-    case 'lg':
-        $maxWidth = 'sm:max-w-lg';
-        break;
-    case 'xl':
-        $maxWidth = 'sm:max-w-xl';
-        break;
-    case '2xl':
-    default:
-        $maxWidth = 'sm:max-w-2xl';
-        break;
-}
+$maxWidth = [
+    'sm' => 'sm:max-w-sm',
+    'md' => 'sm:max-w-md',
+    'lg' => 'sm:max-w-lg',
+    'xl' => 'sm:max-w-xl',
+    '2xl' => 'sm:max-w-2xl',
+][$maxWidth ?? '2xl'];
```

Comment:
Please revert this change, because it's a breaking change.

If someone uses a name that is not defined, he will get an exception:

```blade
<x-jet-modal :maxWidth="XX" />

{{-- ErrorException --}}
{{-- Undefined array key "XX" --}}
```